.ci/install/isal.sh

```diff
@@ -2,15 +2,22 @@
 
 set -ex
 
-ISAL_MIRROR=http://kevinlin.web.rice.edu/static/isal.tar.gz
+ISAL_MIRROR=http://kevinlin.web.rice.edu/static/isal-2.tar.gz
 
 cd $HOME
-if [ ! -d /tmp/cache/isal ]; then
-    wget --quiet -O isal.tar.gz $ISAL_MIRROR
-    tar -xf isal.tar.gz -C /tmp
-    mv /tmp/isa-l_open_src_2.13 /tmp/cache/isal
+if [ -d /tmp/cache/isal ]; then
```

Comment:
This step isn't necessary--we can clear the build cache from Travis directly. If you don't want ISAL to go through the build cache you can just remove this entire `if` block and go here: https://travis-ci.org/comp413-2017/RDFS/caches
and click on "Delete all repository caches"